### PR TITLE
CHAT-9 Lefthand nav - UI Initial

### PR DESCRIPTION
**Pull request by Charleswmm**
Added left hand nav UI for the channel select portion, this includes:

1. Hover state with tooltip
2. Hover colors
3. active state
4. image place holder
5. initials place holder

This story has already been tagged as "Done" on Jira.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,31 +7,96 @@
...
 <body>
 	<div class="app">
-		<nav class="side">
-			<div class="side-server">
-				<div class="side-server-home"></div>
-				<div class="side-server-channel"></div>
-				<div class="side-server-download"></div>
+		<nav class="nav-side">
+			<div class="nav-column nav-column-server">
+				<div class="nav-group nav-group-server">
```
Comment (jakxnz) [inline src/index.html]: This is already in `nav-column-server` so do we need `nav-group-server`, or vise versa?

File: src/index.html [hunk 2]
```diff
@@ -7,31 +7,96 @@
...
+						<button class="btn btn-start">Find or start a conversation</button>
+					</div>
+				</div>
+				<div class="nav-group">
+					<div class="nav-item nav-item-menu">Friends</div>
+					<div class="nav-item nav-item-menu">Nitro</div>
+				</div>
+				<div class="nav-group nav-group-grow">
+					<div class="nav-item-head">
+						DIRECT MESSAGES
```
Comment (jakxnz) [inline src/index.html]: Uppercase characters are often more of an aesthetic thing than a functional thing, so I like to make a rule that all copy will use standard grammar, and use CSS to adjust something like uppercase.

Can we make a rule of always wrapping copy in an element?

File: src/index.html [hunk 3]
```diff
@@ -7,31 +7,96 @@
...
+						<div class="nav-item"></div>
+						<div class="nav-item"></div>
+						<div class="nav-item"></div>
+						<div class="nav-item"></div>
+					</div>
+				</nav>
 			</div>
-			<div class="chat-content"></div>
-			<div class="chat-send"></div>
+			<div class="chat-main"></div>
```
Comment (jakxnz) [inline src/index.html]: `chat-main` is a little ambiguous, could we use something like `chat-body`?

File: src/scss/app.scss [hunk 4]
```diff
@@ -2,19 +2,27 @@
...
-$border-primary: rgba(4,4,5,0.5);
-$interact-hover: #dcddde;
-$interact-active: #fff;
-$interact-muted: #4f545c;
+$text-white: #fff;
+$light-text-grey: #dcddde;
+$lightest-shadow-grey: #8e9297;
+$light-shadow-grey: #4f545c;
+$muted-shadow-grey: #36393f;
+$neutral-shadow-grey: #2f3136;
```
Comment (jakxnz) [inline src/scss/app.scss]: `muted` and `neutral` feel interchangeable. I don't think I would remember which is which when I needed it. Can we differentiate these a little more? Maybe, something like `neutral-shadow-grey` `alt-neutral-shadow-grey`?

File: src/scss/app.scss [hunk 5]
```diff
@@ -24,98 +32,304 @@ $interact-muted: #4f545c;
...
+        background-size: contain;
+        background-repeat: no-repeat;
+        background-position: 0 0.5rem;
+    }
+
+    .svg-add {
+        background-image: url("../img/icons/plus.svg");
+    }
+    .svg-discover {
+            background-image: url("../img/icons/search.svg");
```
Comment (jakxnz) [inline src/scss/app.scss]:
```suggestion
        background-image: url("../img/icons/search.svg");
```

File: src/scss/app.scss [hunk 6]
```diff
@@ -24,98 +32,304 @@ $interact-muted: #4f545c;
...
+    }
+    .svg-download {
+            background-image: url("../img/icons/download.svg");
```
Comment (jakxnz) [inline src/scss/app.scss]:
```suggestion
        background-image: url("../img/icons/download.svg");
```

**Review by jakxnz — CHANGES_REQUESTED** [review]
Looking really good. Thoughtful class names and markup structure, really clean CSS choices.

Could we make a rule to separate CSS sibling selectors with a space? i.e `~` `>` `+` e.g `.parent > .child` (Just to make them easier to spot)

I suspect it's going to be straight forward splitting out `app.scss` into components!

Just a few comments from me...

File: src/index.html (hunk 1 above)
Comment (Charleswmm) [reply src/index.html]: True, nav-group-server is even used.  nav-group-server Removed.

File: src/index.html (hunk 2 above)
Comment (Charleswmm) [reply src/index.html]: Yup, np.

File: src/scss/app.scss (hunk 4 above)
Comment (Charleswmm) [reply src/scss/app.scss]: lol, I struggle to find descriptive names for small color variations. alt works.

**Review by jakxnz — APPROVED** [review]
LGTM

**Pull request merged**